cdc/sink/metrics.go

```diff
@@ -24,7 +24,7 @@ var (
 			Subsystem: "sink",
 			Name:      "txn_batch_size",
 			Help:      "Bucketed histogram of batch size of a txn.",
-			Buckets:   prometheus.ExponentialBuckets(0.0005, 2, 18),
```

Comment:
txn_batch_size is the array length, it's meaningless using `0.0005` and you can only see only about  `66`(relate to the max bucket value) in the dashboard of .99 although all the batch size is thousands.